[docs] add ag-2021 announcement (#5938)
DevExpress/testcafe: docs/articles/blog/2020-2-9-modern-web-testing-with-testcafe.md

---
layout: post
title: "Upcoming Roundtable: Modern Web Testing with TestCafe"
permalink: /media/team-blog/:title.html
isTeamBlog: true
author: TestCafe Team
---
# Upcoming Roundtable: Modern Web Testing with TestCafe

Tomorrow, join the 5th online [Automation Guild conference](https://guildconferences.com/ag-2021/) and learn how to upscale your test automation skill with Dmytro Shpakovskyi, the author of “Modern Web Testing with TestCafe”.

<!--more-->

Dmytro has over a decade of experience in Quality Assurance, Software Testing and Software-as-a-Service industries. Skilled in end-to-end, load, and API test automation, he is certified by the International Software Testing Qualifications Board (ISTQB).

> During his successful career Dmytro led the quality assurance automation for the platform of 60+ highly scalable web applications and services, built and maintained numerous test automation frameworks, managed distributed teams of QA automation engineers, mentored and helped to convert manual test engineers to test automation.

Reviewer: ```suggestion
During his career, Dmytro was in charge of the quality assurance automation of 60+ highly scalable web applications and services, built and maintained numerous test automation frameworks, managed distributed teams of QA automation engineers, and helped engineers to convert to automated testing.
```

> Dmytro creates and open-sources new test automation frameworks, mentors other QA engineers and explores new tools and techniques for automated testing. He shares his experience at [www.stijit.com](https://www.stijit.com/).

Reviewer: ```suggestion
Dmytro creates and open-sources new test automation frameworks, mentors other QA engineers, and explores new tools and techniques for automated testing. He shares his experience at [www.stijit.com](https://www.stijit.com/).
```

> In addition to that, Dmytro authored a book "Modern Web Testing with TestCafe" - a no-nonsense guide to getting started with TestCafe quickly by building a complete test suite while learning the core concepts of test automation with TestCafe (available on [Amazon](https://www.amazon.com/gp/product/B08HM9JXRK)).

Reviewer: ```suggestion
Dmytro is also the author of the book "Modern Web Testing with TestCafe" - a straightforward guide in which he explains how to build a complete test suite while learning the core concepts of test automation with TestCafe (available on [Amazon](https://www.amazon.com/gp/product/B08HM9JXRK)).
```
